### PR TITLE
changes methods and add some comments

### PR DESCRIPTION
**Pull request by Krasavchik-max**

### CODE REVIEW

File: src/main/java/org/example/AllCities.java [hunk 1]
```diff
@@ -1,58 +1,74 @@
...
-        driver.findElement(By.id("mat-input-0")).sendKeys(LGN);
+        driver.findElement(By.id("mat-input-0")).sendKeys(LoginAndPassword.LGN);
         driver.findElement(By.id("mat-input-1")).sendKeys(LoginAndPassword.PSWD);
+
+        //close cookie's window
         driver.findElement(By.xpath("//*[@id=\"onetrust-close-btn-container\"]/button")).click();
-        waitingDriver(5000L);
+        timer(5000L);
+
+        //c
```
Comment (5809909) [inline src/main/java/org/example/AllCities.java:25]: join 2 rows in one

File: src/main/java/org/example/AllCities.java [hunk 2]
```diff
@@ -1,58 +1,74 @@
...
+
+        //click booking button
         driver.findElement(By.xpath("//button[contains(@class,'z-index-999')]")).click();
-        waitingDriver(10000L);
+        timer(10000L);
+
+        //array of cities
         String[] cities = {"Grodno", "Baranovichi", "Brest", "Gomel", "Lida", "Minsk", "Mogilev", "Pinsk"};
 
+        //replace element of cities
```
Comment (5809909) [inline src/main/java/org/example/AllCities.java:37]: //Select city from array

File: src/main/java/org/example/AllCities.java [hunk 3]
```diff
@@ -1,58 +1,74 @@
...
+        timer(10000L);
         driver.findElement(By.xpath("//*[@id=\"mat-select-value-5\"]/span")).click();
-        waitingDriver(2000L);
+        timer(2000L);
         driver.findElement(By.xpath("//span[contains(text(),' Karta Polaka D-visa ')]")).click();
-        waitingDriver(10000L);
+        timer(10000L);
+
+        // print text result
         WebElement webElement = driver.findElement(By.xpath("//div[contains(@class,'alert-info')]"));
```
Comment (5809909) [inline src/main/java/org/example/AllCities.java:72]: webelement is unnecessary element
 String textElement = driver.findElement(By.xpath("//div[contains(@class,'alert-info')]")).getText();

**Review by 5809909 — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/AllCities.java (hunk 2 above)
Comment (Krasavchik-max) [reply src/main/java/org/example/AllCities.java:37]: done

File: src/main/java/org/example/AllCities.java (hunk 3 above)
Comment (Krasavchik-max) [reply src/main/java/org/example/AllCities.java:72]: done

File: src/main/java/org/example/AllCities.java (hunk 1 above)
Comment (Krasavchik-max) [reply src/main/java/org/example/AllCities.java:25]: done

**Review by 5809909 — APPROVED** [review]

**Pull request merged**